src/tools/perf/perftest.c

```diff
@@ -354,6 +354,11 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -p <port>      TCP port to use for data exchange (%d)\n", ctx->port);
 #if HAVE_MPI
     printf("     -P <0|1>       disable/enable MPI mode (%d)\n", ctx->mpi);
+#endif
+#if HAVE_CUDA
+    printf("     -m <mem type>  memory type of messages\n");
+    printf("                        host - system memory(default)\n");
```

Comment:
i guess the mem type argument can be specified even if not HAVE_CUDA. so need to put only the "cuda - NVIDIA GPU memory" in the `#if`